### PR TITLE
Adds layout assessments

### PR DESCRIPTION
**Pull request by argyleink**
3 question pop quiz at the end 👍🏻

### CODE REVIEW

File: src/site/content/en/learn/css/layout/layout.assess.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    options:
+      - content: float
+        rationale: "This is not a display type, it's a property"
+      - content: flex
+        rationale: "Flexbox sizes the boxes and gives them new alignment,
+        distribution and sizing properties"
+      - content: inline-block
+        rationale: "A valid display type, but does not create rules for their
+        children"
+      - content: block
```
Comment (rachelandrew) [inline src/site/content/en/learn/css/layout/layout.assess.yml:33]: I don't think this is right, it defines flow layout for the children https://www.w3.org/TR/css-display-3/#valdef-display-flow

**Review by rachelandrew — CHANGES_REQUESTED** [review]
I'm not sure about question 2 ...

**Comment by argyleink** [comment]
> I'm not sure about question 2 ...

I'll replace it 👍🏻

File: src/site/content/en/learn/css/layout/layout.assess.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    cardinality: "1"
+    correctAnswers: "1"
+    stem: |
+      To flow multiple paragraphs into columns, which CSS property is best for
+      this task?
+    options:
+      - content: "`display: grid`"
+        rationale: "While grid could put multiple paragraphs into columns, those
+        columns would be their own columns, not flowing together from one to the
+        next like is needed."
```
Comment (rachelandrew) [inline src/site/content/en/learn/css/layout/layout.assess.yml:30]:
```suggestion
        rationale: "While grid could put multiple paragraphs into columns, those
        columns would be their own columns, not flowing together from one to the
        next."
```

File: src/site/content/en/learn/css/layout/layout.assess.yml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    stem: "What does it mean if a block is out of flow?"
+    options:
+      - content: "It's stuck on the side of the river"
+        rationale: "Context is CSS here, not geography"
+      - content: "It's been given a `top` or `left` position value"
+        rationale: "Having these properties alone do not pull a box out of flow"
+      - content: "It's no longer positioned based on it's siblings positions"
+        rationale: "A box with `position: absolute` for example, is now
+        positioned with x and y coordinates based off it's containing block, and
+        not it's order with other sibling elements"
```
Comment (rachelandrew) [inline src/site/content/en/learn/css/layout/layout.assess.yml:52]:
```suggestion
        rationale: "Having these properties alone do not pull a box out of flow"
      - content: "It's no longer positioned based on its siblings positions"
        rationale: "A box with `position: absolute` for example, is now
        positioned with x and y coordinates based on the containing block, and
        not its order with other sibling elements"
```

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**